configs/default/FOXE-FOXEERF7V4M6000.config

```diff
@@ -0,0 +1,116 @@
+# Betaflight / STM32F7X2 (S7X2) 4.0.0 Apr  4 2019 / 00:54:12 (e44f75b4a) MSP API: 1.41
+
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC_SPI_MPU6000
+#define USE_BARO_DPS310
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+board_name FOXEERF7V4M6000
```

Comment:
Should we just call it V5 as this looks a lot like V4 + BARO
Or we could add the BARO to V4

Reply:
V4, please, because the board is printed V4 already

Reply:
Okay NP... just add the defines to the V4. No need for a new target :) - will update for 4.5 too.

Reply:
Added the defines, tested, the barometer doesn't work, so add a new target

Reply:
Nah, we are fixing that.

Reply:
@Johnsnow-foxeer be sure to enable expert mode on the firmware flasher tab (it's hidden beneath show release candidates. Then add in custom defines: `ACC_SPI_MPU6000 GYRO_SPI_MPU6000` and flash 4.4.1 (or 4.5)